src/Talis/Persona/Client/Users.php

```diff
@@ -47,13 +51,20 @@ public function getUserByGuids($guids, $token, $cacheTTL = 300)
             throw new \InvalidArgumentException("Invalid token");
         }
 
-        $url = $this->config['persona_host'] . '/users?guids=' . urlencode(implode(',', $guids));
+        $guidsString = implode(',', $guids);
+        $url = $this->config['persona_host'] . '/users?guids=' . urlencode($guidsString);
 
         try {
-            $users = $this->personaGetUser($url, $token, $cacheTTL);
-            return $users;
+            return $this->performRequest(
+                $url,
+                array(
+                    'bearerToken' => $token,
+                    'cacheTTL' => $cacheTTL,
+                )
+            );
         } catch(\Exception $e) {
-            throw new \Exception('User profiles not found');
+            $this->getLogger()->error("Error finding user profiles for guids $guidsString | " . $e->getMessage());
```

Comment:
does this log statement take a second argument? The log frameworks seem to prefer the user to hand in any variable information within the second argument so that it can choose how to display the information.
```php
$this->getLogger()->error(
    "Error finding user profiles for guids",
    [ "guids" => $guidsString, "error" => $e->getMessage() ],
);
```